google_datastream/main.tf

```diff
@@ -0,0 +1,72 @@
+#locals {
+#connection_profile_name = "projects/${var.project_id}/locations/${var.location}/connectionProfiles/${var.source_connection_profile_name}"
+#}
+
+resource "google_datastream_private_connection" "default" {
+  display_name          = "Datastream private connection profile for ${var.application}-${var.realm}-${var.environment}"
+  location              = var.location
+  private_connection_id = "datastream-private-conn-${var.environment}-${var.location}"
+
+  labels = {
+    app_code       = var.application
+    component_code = format("%s-%s", var.application, var.component)
+    env_code       = var.environment
+    realm          = var.realm
+  }
+
+  vpc_peering_config {
+    vpc    = var.vpc_network
+    subnet = var.datastream_subnet
+  }
+}
+
+#resource "google_datastream_connection_profile" "source_connection_profile" {
```

Comment:
How does someone make use of this since it is commented out? Maybe this could put as part of the examples instead?

Reply:
It can't be used unless we want to put the actual db password in terraform. The creation of the source profile and the stream itself have to be done manually :(

Reply:
Okay makes sense. Could we add a comment or details around that in the README or next to the resource definition?